SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialGDKSpanId.cpp

```diff
@@ -31,6 +31,23 @@ const Trace_SpanIdType* FSpatialGDKSpanId::GetConstId() const
 	return Id;
 }
 
+FUserSpanId FSpatialGDKSpanId::ToUserSpan(const FSpatialGDKSpanId& SpanId)
+{
+	FUserSpanId UserSpan;
+	UserSpan.Data.SetNum(sizeof(SpanId));
+	memcpy(UserSpan.Data.GetData(), &SpanId, sizeof(FSpatialGDKSpanId));
```

Comment:
SetNum with sizeof(SpanId) but memcpy with sizeof(FSpatialGDKSpanId) - they are equal but seems asymmetric! Do we static assert this somewhere btw?

Reply:
+1